administrator/components/com_installer/src/Model/DiscoverModel.php

```diff
@@ -305,4 +305,24 @@ protected function getEmptyStateQuery()
 
 		return $query;
 	}
+
+	/**
+	 * Checks uninstalled extensions in extensions table.
+	 *
+	 * @return  boolean  True if there are discovered extensions on the database.
```

Comment:
```suggestion
	 * @return  boolean  True if there are discovered extensions in the database.
```

i appreeciate this was wrong before your pr